vms/platformvm/block/executor/verifier.go

```diff
@@ -605,16 +605,16 @@ func (v *verifier) processStandardTxs(txs []*txs.Tx, feeCalculator txfee.Calcula
 		}
 	}
 
-	// After processing all the transactions, deactivate any SoVs that might not
+	// After processing all the transactions, deactivate any L1 validators that might not
 	// have sufficient fee to pay for the next second.
```

Comment:
Pedantic, but this was wrapping at 80 chars before
```suggestion
	// After processing all the transactions, deactivate any L1 validators that
	// might not have sufficient fee to pay for the next second.
```